dynawo/sources/Models/CPP/Components/ModelLoadRestorativeWithLimits/CMakeLists.txt

```diff
@@ -31,6 +31,11 @@ target_include_directories(${MODEL_NAME} SYSTEM
   $<TARGET_PROPERTY:Boost::boost,INTERFACE_INCLUDE_DIRECTORIES>
   )
 
+target_link_libraries(${MODEL_NAME}
+  PRIVATE
+    Boost::serialization
```

Comment:
this is also a dynawo_CPPModelsCommon dependency but declared private maybe we can put it public to enable the transitivity

Reply:
ok let me try